### ISSUE TITLE
Branch name not displaying with default project "untitled" document

### ISSUE THREAD

**Issue by njpanderson** [atom/status-bar#167]
Atom version: 1.10.0

When opening a project folder without an active session there's an "untitled" document that Atom presents by default. When showing this document, however, there's no branch name in the status bar.

![screen shot 2016-08-31 at 16 38 35](https://cloud.githubusercontent.com/assets/12379690/18135349/8b692a04-6f99-11e6-867c-37c66e772751.png)

Opening a file or closing the "untitled" document resolves this issue.

![screen shot 2016-08-31 at 16 38 46](https://cloud.githubusercontent.com/assets/12379690/18135359/972ee9aa-6f99-11e6-9f46-3f5ee475f709.png)

Easy one to reproduce - just open a project you've not opened before or close all files and use cmd-n to add the "untitled" document.

**Comment by thomasjo** [comment]
This is not a bug; a file that has not been persisted does not belong to a Git repository. For a file to belong to a Git repository it needs to be persisted.

The Git information in the status bar is not related to projects beyond the fact a project is tied to a folder, but since a single Atom workspace/window can have multiple projects, it is not possible to associate new files to a Git repository via a project association.

In summary, this is by design. Thanks for the report though.

**Comment by njpanderson** [comment]
No problem!
